az-digital/az_quickstart
# Closes #1273 Closes #3848: Allow multiple select menus on the same page

## Description
This change makes unique IDs for each select element and adds a reference to that unique ID in the associated label element.

### Release notes
```
This changes the IDs of select menu elements and enhances the accessibility by adding a for element to the markup.
```

## Related Issues
Closes #1273 
Closes #3848 

## How to test
Probo build: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1387.probo.build/dana

1. - [x] Enable the Quickstart select menu module
2. Add two new "select menu" blocks on the same page
3. Operate each block independently
4. Inspect to observe 'for' element on the label and 'id' on the select are the same



### Previous reference
See https://user-images.githubusercontent.com/1023167/169171863-e881d394-b837-46d6-9494-bf07c16b4419.mov



## Types of changes

**Arizona Quickstart** (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

**Drupal core**
- Patch release changes
- Minor release changes

**Drupal contrib projects**
- Patch release changes
- Minor release changes

## Checklist:
